js/src/block-editor/components/edit.js

```diff
@@ -52,12 +89,26 @@ const Edit = ( { blockProps, block } ) => {
 						/>
 					</div>
 				) : (
-					<ServerSideRender
-						block={ `genesis-custom-blocks/${ block.name }` }
-						attributes={ attributes }
-						className="genesis-custom-blocks-editor__ssr"
-						httpMethod="POST"
-					/>
+					<>
+						{
+							hasInnerBlocksField
+								? (
+									<Notice status="info" isDismissible={ false }>
+										{ sprintf(
+											/* translators: %1$s: the field name */
+											__( 'The field %1$s will not display in this preview, but will display on the front-end', 'genesis-custom-blocks' ),
```

Comment:
This is unfortunate, I should have caught it earlier.

`<ServerSideRender>` doesn't send the `InnerBlocks` in the REST request. So it doesn't render the inner blocks.

I couldn't find a workaround, but maybe there is one.

![theblock-sdaf](https://user-images.githubusercontent.com/4063887/123994949-787fe680-d993-11eb-9d0d-77c1f9e17041.png)